apps/snitch_core/lib/core/data/model/payment/payment.ex

```diff
@@ -54,4 +55,18 @@ defmodule Snitch.Data.Model.Payment do
   def to_subtype(%Payment{payment_type: "ccd"} = payment) do
     CardPaymentModel.from_payment(payment.id)
   end
+
+  def compute_order_payments(order) do
```

Comment:
So this function reads like this: `Model.Payment.compute_order_payments/1`.
How about: `Model.Payment.total/1`?